tox.ini

```diff
@@ -11,4 +11,8 @@ deps =
     mock
     pytest
     pytest-cov
-commands = py.test {posargs:-v --cov=pyteamcity --cov-report=term-missing}
+passenv =
+    TEAMCITY_HOST
+    TEAMCITY_USER
+    TEAMCITY_PASSWORD
+commands = py.test {posargs:-v --cov=pyteamcity --cov-report=term-missing test_pyteamcity.py pyteamcity/future/tests/unit/}
```

Comment:
I suggest only having `py.test` here and moving the rest in pytest.ini (and move the path under `testpaths`, not `addopts`). This way one can just type `py.test` without running tox and it will behave the same.